mautic/mautic
# Provide additional API endpoint to clone a specific campaign

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | NO
| New feature? | YES
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
Provide an API endpoint /campaigns/clone/{campaignId} to clone a campaign.

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. 
2. 

#### Steps to test this PR:
1. enable API and applicable authorization method(s)
2. send HTTP request POST /api/campaigns/clone/[campaignId] HTTP/1.1
    Host: [your host here]
3. response should be json serialized campaign object with a new ID
4. Alternatively, test with an invalid campaign ID - response should be "Item not Found"
5. Alternatively, test with a unauthorized account - response should be 403: No Access

#### List deprecations along with the new alternative:
1. 
2. 

#### List backwards compatibility breaks:
1. 
2.
